src/awx_plugins/credentials/azure_kv.py

```diff
@@ -121,11 +121,11 @@ def _initialize_credential(
 def azure_keyvault_backend(  # noqa: WPS211
     *,
     url: str,
-    client: str,
-    secret: str,
-    tenant: str,
+    client: str = '',
+    secret: str = '',
+    tenant: str = '',
     secret_field: str,
```

Comment:
Does it *actually* work with an argument that doesn't have a default in between the others? :thinking:

Reply:
Oh.. That's probably because all these are kw-only.